a11y-meta-display-guide/2.0/draft/techniques/onix-metadata/index.html

```diff
@@ -8,6 +8,7 @@
 		<script class="remove">
 			// <![CDATA[
 			var respecConfig = {
+				shortName: "onix-techniques",
```

Comment:
Shouldn't this short name be more explicit?
Like with a `a11y-display-` prefix?